workflow/local_docker_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# This script can be used to locally check and debug
+# the linux build process outside of CI.
+# This should be run from the root directory `packaging/local_docker_build.sh`
+
+set -e
+
+rm -rf CMakeCache.txt
+find . -name CMakeFiles -print0 | xargs -0 rm -rf
```

Comment:
These don't seem necessary now that we're building out of the source tree. I think the equivalent would be to simply delete `coverage.build` instead.